rio_tiler/io/xarray.py

```diff
@@ -335,17 +335,17 @@ def point(
         ds_lon, ds_lat = transform_coords(coord_crs, self.crs, [lon], [lat])
 
         if not (
-            (self.bounds[0] < ds_lon[0] < self.bounds[2])
-            and (self.bounds[1] < ds_lat[0] < self.bounds[3])
+            (self.bounds[0] < ds_lat[0] < self.bounds[2])
```

Comment:
@benjaminleighton bounds are in form of `xmin, ymin, xmax, ymax` so `self.bounds[0] < ds_lon[0] < self.bounds[2]` is right, I'm not sure why you want to compare latitude (Y) with bounds[0], bounds[1] which are X coordinates?